src/meta-srv/src/procedure/region_failover/activate_region.rs

```diff
@@ -137,6 +141,11 @@ impl ActivateRegion {
                             .context(error::UnexpectedSnafu {
                                 violated: "expected region_options",
                             })?,
+                        self.wal_options_map
+                            .clone()
+                            .context(error::UnexpectedSnafu {
+                                violated: "expected wal_options_map",
+                            })?,
```

Comment:
Forgets to retrieve the `wal options`. It's alway empty here.

Reply:
What do you mean? Can you provide more details?

Reply:
Updated. PTAL.